platforms/tktrex/backend/bin/app.ts

```diff
@@ -87,137 +25,78 @@ export const makeApp = async (
   /* this API is v0 as it is platform neutral. it might be shared among
    * all the trex backends, and should return info on system health, echo OK
    * if the system is OK, and the git log of the code running */
-  app.get(
-    '/api/v0/info',
-    async (req, res) => await iowrapper('systemInfo', req, res)
-  );
```

Comment:
I removed it cause the `systemInfo` handler wasn't present - I spotted it by improve the typings of the `routeHandlerMiddleware` - but we can add those info to the `/api/v0/health`, right?

Reply:
health should just be "OK" (so it is easy for simple monitoring system);

systemInfo can be restored soon or later, in the case we need to know which version is running on a backend. nothing urgent now! 

(back in time, because we had a lot of change in progress, offering git log was seeming helpful)